email/README.md

```diff
@@ -214,3 +214,90 @@ email:
 
 When an email is sent via the `email-test` service above, it will not actually
 be sent. The email contents will be printed on the console.
+
+## Sending an Email to the user an OTP to log in
+
+One of the most common use cases of using an Email Service is to enable 2FA authentication. Here is a small example of how you can setup share OTP using Email with Gramex.
+
+Here are the steps that need to be followed to execute this exercise successfully:-
+
+- Create an event using an object which triggers the event of sending the OTP. This is usually triggered on a click of a button but can be based on any other object.
+- Route the event to an endpoint which triggers the execution of the function which sends the mail.
+- Create a function that fetches the details required like Email ID, OTP number generator from specific database or service, and send a mail.
+
+For this example we need to work on 3 files:-
+
+- gramex.yaml : To create specific endpoints to send mail to a particular address
+- index.html : The HTML page from which the event is fired
+- app.py : Add a Python function to send the Email
+
+**Step 1 - Create a Event to which the OTP should be generated**
+
+```html
+<form action="sendmail" method="POST">
```

Comment:
Let's make this a full-fledged application page, like https://learn.gramener.com/guide/email/mail